src/pages/workspaces/workspace/workflows/DataStepContent.js

```diff
@@ -119,7 +119,7 @@ const DataStepContent = ({
             [chooseRootType, () => rootEntityType],
             [chooseSetType, () => entitySetType]
           ),
-          entityMetadata, setEntityMetadata: () => {}, googleProject, workspaceId: { namespace, name }, workspace, columnDefaults,
+          entityMetadata, setEntityMetadata: () => {}, googleProject, workspaceId: { namespace, name }, workspace,
```

Comment:
Was the passing of `columnDefaults` just a mistake? I don't see that it was removed as an parameter to `DataTable`.

Reply:
It was removed here: https://github.com/DataBiosphere/terra-ui/pull/3030/files#diff-967b453c8b03379166f3e719046b3a65574bddf955e95e14d115bea19185ae3bL87

DataTable destructured its props in multiple places.